profiles/api.py

```diff
@@ -0,0 +1,18 @@
+"""
+RESTful viewsets and serializers for user profiles
+"""
+from rest_framework import serializers, viewsets
+from profiles.models import Profile
+
+
+class ProfileSerializer(serializers.ModelSerializer):
+    """Serializer for Program objects"""
+    class Meta:  # pylint: disable=missing-docstring
+        model = Profile
+        fields = ('id',)
+
+
+class ProfileViewSet(viewsets.ReadOnlyModelViewSet):
+    """API for the Program collection"""
+    queryset = Profile.objects.all()
+    serializer_class = ProfileSerializer
```

Comment:
Ok, we should also change `courses/api.py` to `courses/serializers.py` and `courses/views.py` to be consistent